src/lib/json-rpc.ts

```diff
@@ -7,6 +9,10 @@ export interface JsonRpcRequest {
   id?: JsonRpcId;
   method: string;
   params?: JsonRpcParams;
```

Comment:
For my understanding of the design, why is it that `subscribe/subscription` is separate from `params`?

Reply:
I figured I would stay within the realms of the `JSON RPC` spec, params should be used for params passed into the method, or something representing them. When looking at how people "extend" JSON RPC to have additional functionality, the common practice is to add these properties outside of params and within the greater JsonRpcRequest.

Reply:
I see, so if I understand correctly, you wanted the JSON RPC subscription be a  feature by itself not dependent on DWN.